dpnp/tests/helper.py

```diff
@@ -464,6 +464,14 @@ def is_lts_driver(device=None):
     return dev.has_aspect_gpu and "1.3" in dev.driver_version
 
 
+def is_dg2(device=None):
+    """
+    Return True if a test is running on DG2 (Intel Arc Alchemist family) GPU device,
+    False otherwise.
+    """
+    return _get_dev_mask(device) in (0x4F00, 0x5600)
```

Comment:
The affected tests are running on another device, so the solution seems [not working](https://cje-fm-owrp-prod04.devtools.intel.com/satg-dap-intelpython/job/intel-packages/job/dpnp/job/dev-windows-py3.13/job/test/247/cloudbees-pipeline-explorer/).